autodocodec-api-usage/test_resources/yaml-schema/void.txt

```diff
@@ -0,0 +1 @@
+#[32many of [][m
```

Comment:
This is the only way (I know of) to represent `Void` in JSONSchema: it doesn't seem to be "illegal", but rather referred to as an "illogical schema".